modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ContentSectionDisplayContext.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.site.cms.site.initializer.internal.display.context;
+
+import com.liferay.frontend.data.set.model.FDSActionDropdownItem;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItem;
+import com.liferay.petra.string.StringBundler;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Sam Ziemer
+ */
+public class ContentSectionDisplayContext {
+
+	public String getAPIURL() {
+		StringBundler sb = new StringBundler();
+
+		sb.append("/o/search/v1.0/search?emptySearch=true");
+		sb.append("&nestedFields=embedded&entryClassNames=");
+
+		for (int i = 0; i < _CONTENT_CLASS_NAMES.length; i++) {
```

Comment:
@SamZiemer, instead of the for, you can use:  `ArrayUtil.toString(_CONTENT_CLASS_NAMES, StringPool.BLANK); `